### ISSUE TITLE
GitHub Action to compile/verify nef Playground

### ISSUE THREAD

**Issue by miguelangel-dev** [bow-swift/nef#135]
We want to publish a GitHub action to let users compile its `nef Playgrounds` (with 3rd-party libraries)

**Comment by bitomule** [comment]
I did a test with steps that may be helpfull on creating the github action.

I added this steps:

- Install homebrew
- Install nef
- Run nef generate md (nef markdown --project Playground.app/  --output /)

And it failed with "Process completed with exit code 255"

```
Run nef markdown --project Playground.app/  --output /
Get playgrounds in 'Playground'... ✓
Get pages in playground 'Playground'... ✓
	• Processing page Readme... ✓
	• Processing page Example... ✓
	• Processing page Testing... ✓
☠️ rendering Xcode Playgrounds from 'Playground.app'
##[error]Process completed with exit code 255.
```

Any ideas?

**Comment by miguelangel-dev** [comment]
Ey! Great @bitomule: maybe you do not have write-permissions in `/`, try to use `. ` or a different path.

On another hand, I have started to work on this, but right now I have stopped, but maybe you can test it and continue from that point - I will invite you to the repo (it is in an experimental phase).

**Comment by bitomule** [comment]
Yeah, changing to . worked, my mistake. Right now I have an action generating md files and a step that even moves readme to main repo README. This is crazy useful to have playgrounds and docs in projects generated in github for free.

**Comment by miguelangel-dev** [comment]
I am glad it is helpful! Yep, that's the idea, keep your docs verified (you can compile the Playground before to generate the markdown files) and updated! ^^

When you integrate `nef` to your CI it starts to be awesome!
